fix(test): adjust flatpak setup test to fix failures

HastD opened the pull request:
If the service is running, sleep a few more seconds after checking service status to give it time to set up the flathub-verified remote.

RoyalOughtness commented on .github/workflows/integration_tests/check_flatpak_setup.sh:
```diff
@@ -40,13 +40,13 @@ check-installed-flatpaks() {
 fi
 
+# Wait a few seconds to give the service time to start
+sleep 5
```
could also sleep for longer

HastD replied:
Unless we want to modify the timer to have a lower [AccuracySec](https://www.freedesktop.org/software/systemd/man/latest/systemd.timer.html#AccuracySec=) (which has power usage implications since it impacts how frequently CPU wakeups for timer scheduling are required), we'd have to sleep a full minute to guarantee that the timer will have started by then, and that seems like an unreasonably long time to sleep for a single integration test like this.

RoyalOughtness replied:
@HastD sleeping for a minute seems reasonable to be able to consistently test this

HastD replied:
Actually, I have another idea, going to test something out on my fork first.

HastD replied:
Okay, it works the way I thought: if we sleep a few more seconds *after* checking the service status, then it gives the service the time it needs to run. I think what's going on is that the act of checking the service status itself triggers a wakeup that causes a scheduled timer to activate, which is why the activation time kept being within a fraction of a second of when the test ran.

HastD committed:
fix(test): adjust flatpak setup test to fix failures

If the service is running, sleep a few more seconds after checking
service status to give it time to set up the flathub-verified remote.

Signed-off-by: Daniel Hast <hast.daniel@protonmail.com>

RoyalOughtness committed:
Merge branch 'live' into fix-integration-test

RoyalOughtness committed:
Merge branch 'live' into fix-integration-test
